README.md

```diff
@@ -4,7 +4,7 @@ The CustomIndexBundle allows create index for doctrine entities using annotation
 
 ## Installation
 
-CustomIndexBundle requires Symfony 5 or higher. Work only with PostgreSQL.
+CustomIndexBundle requires Symfony 5 or higher. Works only with PostgreSQL.
 
 Run into your project directory:
 ```
```

Comment:
Code below is outdated. I thinks we should upgrade it to php8 (return type, short array syntax)

Reply:
updated